src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -171,6 +171,10 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs):  # pyli
 register_cli_argument('functionapp create', 'plan', options_list=('--plan', '-p'), completer=get_resource_name_completion_list('Microsoft.Web/serverFarms'),
                       help="name or resource id of the function app service plan. Use 'appservice plan create' to get one")
 register_cli_argument('functionapp create', 'consumption_plan_location', options_list=('--consumption-plan-location', '-c'),
-                      help="Geo location for functionsapps with consumption plans. Use 'functionapp list-consumption-locations' to see possible location plans")
+                      help="Geographic location where Function App will be hosted. Use 'functionapp list-consumption-locations' to view available locations.")
 register_cli_argument('functionapp create', 'storage_account', options_list=('--storage-account', '-s'),
                       help='Provide a string value of a Storage Account in the provided Resource Group. Or Resource ID of a Storage Account in a different Resource Group')
+
+register_cli_argument('functionapp config appsettings', 'settings', nargs='+', help="space separated app settings in a format of <name>=<value>")
```

Comment:
@aspaan, in the next PR, please do Travis [suggested in the previous PR ](https://github.com/Azure/azure-cli/pull/2829/files#r110995725)